### PR TITLE
new(packaging): nokia (formerly alcatel) isam snmp plugin

### PR DESCRIPTION
**Pull request by omercier**
## Description

The plugin has been there for a while but never packaged.

REFS: CTOR-594

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Functionality enhancement or optimization (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## How this pull request can be tested ?

No data for tests right now.

## Checklist

- [x] I have followed the **[coding style guidelines](https://github.com/centreon/centreon-plugins/blob/develop/doc/en/developer/plugins_global.md#5-code-style-guidelines)** provided by Centreon
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (develop).
- [ ] I have implemented automated tests related to my commits.
- [x] I have reviewed all the help messages in all the .pm files I have modified.
  - [x] All sentences begin with a capital letter.
  - [x] All sentences are terminated by a period.
  - [x] I am able to understand all the help messages, if not, exchange with the PO or TW to rewrite them.

### CODE REVIEW

File: packaging/centreon-plugin-Network-Nokia-Isam-Snmp/pkg.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "pkg_name": "centreon-plugin-Network-Nokia-Isam-Snmp",
+    "pkg_summary": "Centreon Plugin to monitor Nokia (formerly Alcatel) Isam devices",
+    "plugin_name": "centreon_nokia_isam_snmp.pl",
+    "files": [
+        "centreon/plugins/script_snmp.pm",
+        "centreon/plugins/snmp.pm",
+        "snmp_standard/mode/interfaces.pm",
+        "snmp_standard/mode/listinterfaces.pm",
+	    "snmp_standard/mode/resources/",
```
Comment (lucie-dubrunfaut) [inline packaging/centreon-plugin-Network-Nokia-Isam-Snmp/pkg.json:10]:
```suggestion
        "snmp_standard/mode/resources/",
```

**Review by lucie-dubrunfaut — DISMISSED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

File: tests/resources/spellcheck/stopwords.t [hunk 2]
```diff
@@ -110,3 +110,9 @@ uptime
 userpass
 v1
 v2
+sfp.temperature
```
Comment (Evan-Adam) [inline tests/resources/spellcheck/stopwords.t:113]: Please keep the alphabetical order of the file

File: src/network/nokia/isam/snmp/mode/hardware.pm [hunk 3]
```diff
@@ -88,20 +88,19 @@ You can also exclude items from specific instances: --filter=cardtemperature,108
...
 
-Set warning threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the warning thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
 Example: --warning='sfp.temperature,1.1,30'
 
 =item B<--critical>
 
-Set critical threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the critical thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
 Example: --warning='sfp.temperature,1.1,40'
```
Comment (Evan-Adam) [inline src/network/nokia/isam/snmp/mode/hardware.pm:104]: I don't anderstand this exemple, the regex is 1.1 ? there is no catching group in the regexp ?

**Review by Evan-Adam — CHANGES_REQUESTED** [review]

File: src/network/nokia/isam/snmp/mode/hardware.pm (hunk 3 above)
Comment (omercier) [inline src/network/nokia/isam/snmp/mode/hardware.pm:104]:
```suggestion
Example: --critical='sfp.temperature,1\.1,40'
```

File: src/network/nokia/isam/snmp/mode/hardware.pm [hunk 4]
```diff
@@ -88,20 +88,19 @@ You can also exclude items from specific instances: --filter=cardtemperature,108
...
 =item B<--warning>
 
-Set warning threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the warning thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
 Example: --warning='sfp.temperature,1.1,30'
 
 =item B<--critical>
 
-Set critical threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the critical thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
```
Comment (omercier) [inline src/network/nokia/isam/snmp/mode/hardware.pm:103]:
```suggestion
Define the critical thresholds for an instance of a certain type of sensor.
Syntax: <type>,<regexp>,<threshold>.
Supported types of sensors: 'sfp.temperature', 'cardtemperature'.
Instances are identified by the last two parts of the OID, separated by a period (example: 1.1) and are filtered as a regular expression.
```

File: src/network/nokia/isam/snmp/mode/hardware.pm [hunk 5]
```diff
@@ -88,20 +88,19 @@ You can also exclude items from specific instances: --filter=cardtemperature,108
...
-
 =item B<--threshold-overload>
 
 Use this option to override the status returned by the plugin when the status label matches a regular expression (syntax: section,[instance,]status,regexp).
 Example: --threshold-overload='sfp,OK,^los$'
 
 =item B<--warning>
 
-Set warning threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the warning thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
```
Comment (omercier) [inline src/network/nokia/isam/snmp/mode/hardware.pm:98]:
```suggestion
Define the warning thresholds for an instance of a certain type of sensor.
Syntax: <type>,<regexp>,<threshold>.
Supported types of sensors: 'sfp.temperature', 'cardtemperature'.
Instances are identified by the last two parts of the OID, separated by a period (example: 1.1) and are filtered as a regular expression.
```

File: src/network/nokia/isam/snmp/mode/hardware.pm [hunk 6]
```diff
@@ -88,20 +88,19 @@ You can also exclude items from specific instances: --filter=cardtemperature,108
...
 Example: --warning='sfp.temperature,1.1,30'
```
Comment (omercier) [inline src/network/nokia/isam/snmp/mode/hardware.pm:99]:
```suggestion
Example: --warning='sfp.temperature,1\.1,30'
```

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by Evan-Adam — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**